### PR TITLE
Use double quotes

### PR DESCRIPTION
**Pull request by GustavoLCR**
This fixes failing tests in Windows

### CODE REVIEW

File: new/db/formats/pe/dep [hunk 1]
```diff
@@ -15,7 +15,7 @@ true
 
 EOF
 CMDS=<<EOF
-!rabin2 -k 'pe.nx' ../bins/pe/dep.exe
+!rabin2 -k "pe.nx" ../bins/pe/dep.exe
```
Comment (radare) [inline new/db/formats/pe/dep]: no need for quotes here

File: new/db/formats/pe/no_dep [hunk 2]
```diff
@@ -15,7 +15,7 @@ false
 
 EOF
 CMDS=<<EOF
-!rabin2 -k 'pe.nx' ../bins/pe/no_dep.exe
+!rabin2 -k "pe.nx" ../bins/pe/no_dep.exe
```
Comment (radare) [inline new/db/formats/pe/no_dep]: same

File: new/db/cmd/md5sha1 [hunk 3]
```diff
@@ -31,9 +31,9 @@ EOF
 CMDS=<<EOF
 b 12
 w hello world
-!rahash2 -qa crc16 -s 'hello world'
+!rahash2 -qa crc16 -s "hello world"
```
Comment (radare) [inline new/db/cmd/md5sha1]: why?

Comment (GustavoLCR) [reply new/db/cmd/md5sha1]: single quotes doesn't work on windows, so the input becomes `'hello` and the tests fails

**Pull request merged**